### ISSUE TITLE
[COLOR] Disability-benefits all-claims has axe violations

### ISSUE THREAD

**Issue by bkjohnson** [department-of-veterans-affairs/va.gov-team#14013]
## [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

This is a follow-up for #13647

All-claims has a few [`serious` axe violations](http://jenkins.vfs.va.gov/blue/organizations/jenkins/testing%2Fvets-website/detail/13647-disability-benefits/10/pipeline/) under the stricter Cypress axe checks.

```
[2020-09-24T16:10:50.624Z]   Running:  [90mapplications/disability-benefits/all-claims/tests/all-claims.cypress[39m           [90m(5 of 51)[39m
[2020-09-24T16:10:50.624Z]             [90m.spec.js[39m                                                                                
[2020-09-24T16:10:51.545Z] [info] [webpackbar] Compiling Webpack
[2020-09-24T16:10:56.775Z] [success] [webpackbar] Webpack: Compiled successfully in 4.97s
[2020-09-24T16:11:00.926Z] 
[2020-09-24T16:11:00.926Z] [0m[0m
[2020-09-24T16:11:00.926Z] [0m  21-526EZ disability compensation claim form[0m
[2020-09-24T16:11:00.926Z] [0m    full-781-781a-8940-test.json[0m
[2020-09-24T16:12:08.534Z]     [32m  ✓[0m[90m fills the form[0m[31m (67196ms)[0m
[2020-09-24T16:12:08.534Z] [0m    maximal-test[0m
[2020-09-24T16:12:20.682Z] 1 accessibility violation was detected
[2020-09-24T16:12:20.682Z] ┌─────────┬──────────────────┬───────────┬───────────────────────────────────────────────────────────────────────────────────────────────────────────┬───────┐
[2020-09-24T16:12:20.682Z] │ (index) │        id        │  impact   │                                                description                                                │ nodes │
[2020-09-24T16:12:20.682Z] ├─────────┼──────────────────┼───────────┼───────────────────────────────────────────────────────────────────────────────────────────────────────────┼───────┤
[2020-09-24T16:12:20.682Z] │    0    │ 'color-contrast' │ 'serious' │ 'Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds' │   3   │
[2020-09-24T16:12:20.682Z] └─────────┴──────────────────┴───────────┴───────────────────────────────────────────────────────────────────────────────────────────────────────────┴───────┘
[2020-09-24T16:12:22.040Z]     [31m  1) fills the form[0m
[2020-09-24T16:12:22.040Z] [0m    maximal-bdd-test[0m
[2020-09-24T16:12:22.040Z]     [36m  - fills the form[0m
[2020-09-24T16:12:22.040Z] [0m    minimal-test[0m
[2020-09-24T16:12:36.860Z]     [32m  ✓[0m[90m fills the form[0m[31m (13299ms)[0m
[2020-09-24T16:12:36.860Z] [0m    minimal-bdd-test[0m
[2020-09-24T16:12:36.860Z]     [36m  - fills the form[0m
[2020-09-24T16:12:36.860Z] [0m    newOnly-test[0m
[2020-09-24T16:12:46.780Z] 1 accessibility violation was detected
[2020-09-24T16:12:46.780Z] ┌─────────┬──────────────────┬───────────┬───────────────────────────────────────────────────────────────────────────────────────────────────────────┬───────┐
[2020-09-24T16:12:46.781Z] │ (index) │        id        │  impact   │                                                description                                                │ nodes │
[2020-09-24T16:12:46.781Z] ├─────────┼──────────────────┼───────────┼───────────────────────────────────────────────────────────────────────────────────────────────────────────┼───────┤
[2020-09-24T16:12:46.781Z] │    0    │ 'color-contrast' │ 'serious' │ 'Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds' │   3   │
[2020-09-24T16:12:46.781Z] └─────────┴──────────────────┴───────────┴───────────────────────────────────────────────────────────────────────────────────────────────────────────┴───────┘
[2020-09-24T16:12:49.286Z]     [31m  2) fills the form[0m
[2020-09-24T16:12:49.286Z] [0m    secondary-new-test.json[0m
[2020-09-24T16:13:01.467Z] 1 accessibility violation was detected
[2020-09-24T16:13:01.467Z] ┌─────────┬──────────────────┬───────────┬───────────────────────────────────────────────────────────────────────────────────────────────────────────┬───────┐
[2020-09-24T16:13:01.467Z] │ (index) │        id        │  impact   │                                                description                                                │ nodes │
[2020-09-24T16:13:01.467Z] ├─────────┼──────────────────┼───────────┼───────────────────────────────────────────────────────────────────────────────────────────────────────────┼───────┤
[2020-09-24T16:13:01.467Z] │    0    │ 'color-contrast' │ 'serious' │ 'Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds' │   1   │
[2020-09-24T16:13:01.467Z] └─────────┴──────────────────┴───────────┴───────────────────────────────────────────────────────────────────────────────────────────────────────────┴───────┘
[2020-09-24T16:13:04.749Z]     [31m  3) fills the form[0m
[2020-09-24T16:13:04.749Z] [0m    upload-781-781a-8940-test.json[0m
[2020-09-24T16:13:22.769Z]     [32m  ✓[0m[90m fills the form[0m[31m (18407ms)[0m
[2020-09-24T16:13:22.769Z] 
[2020-09-24T16:13:22.769Z] 
[2020-09-24T16:13:22.769Z] [92m [0m[32m 3 passing[0m[90m (2m)[0m
[2020-09-24T16:13:22.769Z] [36m [0m[36m 2 pending[0m
[2020-09-24T16:13:22.769Z] [31m  3 failing[0m
```

The fix can be continued on this branch: https://github.com/department-of-veterans-affairs/vets-website/compare/13647-disability-benefits

## Acceptance Criteria

- [ ] The temporary _13647Exception is removed from disability-benefits Cypress tests
- [ ] disability-benefits passes all Cypress axe checks

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

**Comment by 1Copenut** [comment]
@bkjohnson LMK if the team is having trouble getting these color contrast issues to appear when I'm stepping through the form and running the axe browser plugin. It *might* be a false positive. In that case, we could ignore the color contrast rule, but I'll hold that until the team has had a chance to investigate.

**Comment by luke-gcio** [comment]
Plan is to attempt fix in sprint 33 still.

**Comment by bkjohnson** [comment]
@Mottie I'm reopening this since the issue still exists as of today:

![image](https://user-images.githubusercontent.com/2008881/105910764-403c3c80-5fde-11eb-8a28-24f182a78ec9.png)

**Comment by Mottie** [comment]
Thanks! I'll follow up on this after axe-core is updated to v4.1+

**Comment by Mottie** [comment]
False-positive color contrast issues are still showing up in axe... still waiting for https://github.com/department-of-veterans-affairs/va.gov-team/issues/16544 to get completed.

**Comment by 1Copenut** [comment]
@Mottie I shipped a fix for #16544 on 2/19. Color contrast rule is disabled by default, can be turned back on by passing two arguments to `cy.axeCheck()`:

```javascript
cy.axeCheck(
  'main',
  {
    rules: {
      'color-contrast': { enabled: true },
    },
  }
);
```

Please see draft PR https://github.com/department-of-veterans-affairs/va.gov-team/pull/20058 for the documentation update I'm working on that further documents this change and links back to the relevant axe-core API.

**Comment by Mottie** [comment]
Done!
